pkg/asset/machines/azure/machinesets.go

```diff
@@ -28,13 +28,12 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 		total = int32(*pool.Replicas)
 	}
 
-	// TODO(flaper87): Add support for availability zones
 	var machinesets []*clusterapi.MachineSet
 	provider, err := provider(platform, mpool, osImage, userDataSecret)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create provider")
 	}
-	// TODO(flaper87): Implement AZ support sometime soon
+	// TODO(serbrech): Implement AZ support
```

Comment:
this change is in the wrong commit.

Reply:
Moved it. you're pushing my git-fu there 🥋